mutation_partition_view.cc

```diff
@@ -45,6 +45,10 @@
 
 using namespace db;
 
+GCC6_CONCEPT(static_assert(MutationViewVisitor<mutation_partition_view_virtual_visitor>);)
```

Comment:
Not suggesting this is wrong or has to be changed, it is just strange to see a concept "floating" like that. When will this be checked? When the TU is compiled?

Reply:
Yes. I don't want to run the check in the header, where it will be checked many times.